src/types.ts

```diff
@@ -49,6 +49,8 @@ export interface LikeCoinWalletConnectorConfig {
   isShowMobileWarning?: boolean;
 
   availableMethods?: LikeCoinWalletConnectorMethodType[];
+
+  defaultLanguage?: string;
```

Comment:
Should this be called `lanauge` instead of `defaultLanguage`? Callers  are explicitly setting it anyway

Reply:
Updated